dbsync-central/release_stuff/application.properties

```diff
@@ -0,0 +1,191 @@
+
+# User name of the openMRS datasource
+spring.openmrs-datasource.username=${spring_openmrs_datasource_user}
+
+# Password of the openMRS datasource
+spring.openmrs-datasource.password=${spring_openmrs_datasource_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+# *********************** Configuration of the dbsync mgt database to write to *******************************************
+#
+# The IP address or host name of the dbsync mgt MySQL database
+dbsync.db.host=${dbsync_db_host}
+
+# The dbsync mgt MySQL database port
+dbsync.db.port=${dbsync_db_port}
+
+# The dbsync mgt MySQL database name
+dbsync.db.name=${dbsync_db_name}
+
+
+# *********************** Configuration of the Management database *****************************************************
+#
+# Driver class of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.driverClassName=com.mysql.cj.jdbc.Driver
+
+# Dialect of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.dialect=org.hibernate.dialect.MySQLDialect
+
+# Url of the management datasource
+spring.mngt-datasource.jdbcUrl=jdbc:mysql://${dbsync.db.host}:${dbsync.db.port}/${dbsync_db_name}?createDatabaseIfNotExist=true
+
+# User name of the management datasource
+spring.mngt-datasource.username=${spring_openmrs_datasource_user}
+
+# Password of the management datasource
+spring.mngt-datasource.password=${spring_openmrs_datasource_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** ActiveMQ configuration ***********************************************************************
+#
+# Broker URL of the Artemis MQ broker shared with the receiver (SHOULD include the protocol e.g. tcp://192.168.1.2:61616)
+#spring.artemis.brokerUrl=tcp://artemis:61616
+#
+spring.artemis.host=${spring_artemis_host}
+spring.artemis.port=${spring_artemis_port}
+# User name of the Artemis broker
+spring.artemis.user=${spring_artemis_user}
+#
+# Password of the Artemis broker
+spring.artemis.password=${spring_artemis_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the ArtemisMQ endpoint to read from inbound DB sync payloads ****************
+# Use either one of the following confs. JMS endpoint if activeMQ is configured or file endpoint for TESTING purposes
+# Camel JMS Endpoint e.g. activemq:topic:<topic-name> or FILE Endpoint e.g. file:<folderPath> Should be absolute path
+camel.input.endpoint=activemq:topic:openmrs.sync.topic?subscriptionDurable=true&durableSubscriptionName=DB-SYNC-RECEIVER&connectionFactory=activeMqConnFactory&acknowledgementModeName=CLIENT_ACKNOWLEDGE&messageListenerContainerFactory=customMessageListenerContainerFactory
```

Comment:
@JPboane please make this configurable, should default to queue and not topic so that for the new provinces we centralize we default to queue, since Zambezia is an existing one it's hard to change it so it is the only one we should keep using a topic, we find away to switch it too later.

Reply:
It also means you need to update the remote configuration to read from the queue instead of topic, so the queue name needs to be changed from **openmrs.sync.topic** to something like **openmrs.sync**

Reply:
This is because we disabled auto queue deletion [here](https://github.com/FriendsInGlobalHealth/centralization-docker-setup/blob/EC-113/artemis/broker.xml#L165) which addresses the bug that forced us to use a topic instead of a queue. Queues are easier to use since there is no requirement for the receiver to always connect  first to establish a subscription.